matchbox_demo/src/main.rs

```diff
@@ -189,17 +187,21 @@ fn lobby_system(
         .start_p2p_session(socket)
         .expect("failed to start session");
 
-    commands.insert_resource(sess);
-    commands.insert_resource(SessionType::P2PSession);
+    commands.insert_resource(Session::P2PSession(sess));
 
     // transition to in-game state
     app_state
         .set(AppState::InGame)
         .expect("Tried to go in-game while already in-game");
 }
 
-fn log_ggrs_events(mut session: ResMut<P2PSession<GGRSConfig>>) {
-    for event in session.events() {
-        info!("GGRS Event: {:?}", event);
+fn log_ggrs_events(mut session: ResMut<Session<GGRSConfig>>) {
+    match session.as_mut() {
+        Session::P2PSession(s) => {
+            for event in s.events() {
+                println!("GGRS Event: {:?}", event);
```

Comment:
```suggestion
                info!("GGRS Event: {:?}", event);
```
Otherwise we'll break logging on wasm.